bindings/java/src/main/java/org/openucx/jucx/examples/UcxReadBWBenchmarkSender.java

```diff
@@ -28,6 +26,13 @@ public static void main(String[] args) throws Exception {
         String serverHost = argsMap.get("s");
         UcpEndpoint endpoint = worker.newEndpoint(new UcpEndpointParams()
             .setPeerErrorHandlingMode()
+            .setErrorHandler((ep, status, errorMsg) -> {
+                if (status == -25) {
```

Comment:
`UCS_ERR_CONNECTION_RESET` instead of `-25`?

Reply:
unfortunately Java API doesn't  define status constants, @petro-rudenko , right?

Reply:
Fixed